src/web/mjs/connectors/DrakeScans.mjs

```diff
@@ -1,12 +1,12 @@
-import WordPressMadara from './templates/WordPressMadara.mjs';
-
-export default class DrakeScans extends WordPressMadara {
+import WordPressMangastream from './templates/WordPressMangastream.mjs';
 
+export default class MonzeeKomik extends WordPressMangastream {

```

Comment:
Its not Monzeekomik ;)

Reply:
Drake scan website dosen't uses the wordpressmadara type anymore it uses one similar to monzeekomik,  so it has to change

reason is, it  dosen't display mangas anymore, so obviously it dosen't download any new chapters
after did the changes it worked can you please check

Reply:
Drake scan website dosen't uses the wordpressmadara type anymore it uses one similar to monzeekomik,  so it has to change

reason is, it  dosen't display mangas anymore, so obviously it dosen't download any new chapters
after did the changes it worked can you please check

Reply:
@MikeZeDev drake scans are using a similar template, so obviously i changed the template to match its new one, and it now works

Reply:
Your class name is wrong. It needs to be matching the file name, change it.

Reply:
Stop spamming the same useless screenshot and fix what needs to be fixed 🙏

Reply:
i apologize i didn't noticed that, the changes i have here is different let me fix that and change it

Reply:
@MikeZeDev done